filebeat/prospector/prospector.go

```diff
@@ -21,6 +21,7 @@ var (
 )
 
 type Prospector struct {
+	harvesterCounter uint64         // Must be 8-byte aligned. Ensured if first field in struct
```

Comment:
Thanks for the feedback @ruflin. Should I make the changelog entry under `Added` or `Bugfixes`? I know you wrote `Added` above but thought I'd double check since this is a bug fix.